pyo3-build-config/src/lib.rs

```diff
@@ -15,7 +15,12 @@ use std::io::Cursor;
 #[cfg(feature = "resolve-config")]
 use once_cell::sync::OnceCell;
 
-pub use impl_::{BuildFlag, BuildFlags, InterpreterConfig, PythonImplementation, PythonVersion};
+pub use impl_::{
+    BuildFlag, BuildFlags, CrossCompileConfig, InterpreterConfig, PythonImplementation,
+    PythonVersion,
+};
+
+pub use impl_::{cross_compiling, find_sysconfigdata, parse_sysconfigdata};
```

Comment:
I think these can just be a single `pub use` rather than splitting the functions tbh.